### PR TITLE
Improved content for Tutorial 7(QAMP fall'22)

### PR DESCRIPTION
**Pull request by SanyaNanda**
### Summary
Enhancing the documentation of Tutorial "07_pegasos_qsvc" as a mentee for QAMP 2022.
Mentor: @ElePT
Mentee: @SanyaNanda

### Details and comments
Restructured the notebook (Overview, Introduction, Objective, Tutorial, accuracy table and what was learned)
Imports placed where required
Added more explanations and inline comments for better understanding of the code
Hyperlinks to API references wherever required

### CODE REVIEW

File: docs/tutorials/07_pegasos_qsvc.ipynb [hunk 1]
```diff
@@ -7,9 +7,13 @@
    "source": [
     "# Pegasos Quantum Support Vector Classifier\n",
     "\n",
-    "There's another SVM based algorithm that benefits from the quantum kernel method. Here, we introduce an implementation of a another classification algorithm, which is an alternative version to the `QSVC` available in Qiskit Machine Learning and shown in the [\"Quantum Kernel Machine Learning\"](./03_quantum_kernel.ipynb) tutorial. This classification algorithm implements the Pegasos algorithm from the paper \"Pegasos: Primal Estimated sub-GrAdient SOlver for SVM\" by Shalev-Shwartz et al., see: https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf.\n",
+    "## Overview\n",
+    "In this tutorial, we will implement another SVM based algorithm that benefits from the quantum kernel method, namely `PegasosQSVC` from Qiskit Machine Learning.\n",
```
Comment (ElePT) [inline docs/tutorials/07_pegasos_qsvc.ipynb:11]: I would suggest to always use either Qiskit Machine Learning or `qiskit-machine-learning` (leaning against the second one)

File: docs/tutorials/07_pegasos_qsvc.ipynb [hunk 2]
```diff
@@ -7,9 +7,13 @@
...
     "\n",
-    "This algorithm is an alternative to the dual optimization from the `scikit-learn` package, benefits from the kernel trick, and yields a training complexity that is independent of the size of the training set. Thus, the `PegasosQSVC` is expected to train faster than QSVC for sufficiently large training sets.\n",
+    "## Introduction\n",
+    "Here, we are demonstrating another approach to classification that implements the Pegasos algorithm from the paper [Pegasos: Primal Estimated sub-GrAdient SOlver for SVM](https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf) by Shalev-Shwartz et al. We introduce the implementation of the classification algorithm, `PegasosQSVC`, as an alternative to the `QSVC` available in Qiskit Machine Learning. For a refresher on QSVC, revisit [\"Quantum Kernel Machine Learning\"](./03_quantum_kernel.ipynb) tutorial. \n",
```
Comment (ElePT) [inline docs/tutorials/07_pegasos_qsvc.ipynb:14]:
```suggestion
    "Here, we are demonstrating another approach to classification that implements the Pegasos algorithm from the paper [Pegasos: Primal Estimated sub-GrAdient SOlver for SVM](https://home.ttic.edu/~nati/Publications/PegasosMPB.pdf) by Shalev-Shwartz et al. We introduce the implementation of the classification algorithm, `PegasosQSVC`, as an alternative to the `QSVC` available in Qiskit Machine Learning. For a refresher on `QSVC`, you can revisit the [\"Quantum Kernel Machine Learning\"](./03_quantum_kernel.ipynb) tutorial. \n",
```

File: docs/tutorials/07_pegasos_qsvc.ipynb [hunk 3]
```diff
@@ -19,59 +23,185 @@
    "id": "thirty-painting",
    "metadata": {},
    "source": [
-    "Let's generate some data:"
+    "### 1. Preparing dataset"
```
Comment (ElePT) [inline docs/tutorials/07_pegasos_qsvc.ipynb:26]: I would add a section with "the rest of the tutorial is structured as follows": index

**Review by ElePT — COMMENTED** [review]
Hello! here are some initial review comments, I will add more later.

File: docs/tutorials/07_pegasos_qsvc.ipynb (hunk 3 above)
Comment (SanyaNanda) [reply docs/tutorials/07_pegasos_qsvc.ipynb]: sure

**Comment by oscar-wallis** [comment]
Closing this PR whilst working on updating tutorials. For more info - see [here](https://github.com/qiskit-community/qiskit-machine-learning/pull/491#issuecomment-1979205893).

**Pull request closed without merging**